pysatNASA/instruments/iss_fpmu.py

```diff
@@ -78,7 +80,7 @@ def init(self):
 
 
 def clean(self):
-    """Return FPMU data cleaned to the specified level.
+    """Clean ISS FPMU data to the specified level.
```

Comment:
Recommend moving the comments below above the code line.

Reply:
?

Reply:
A couple code lines below have
```
variable = stuff  # stuff happened
```

but we normally have done
```
# stuff is gonna happen
variable = stuff
```